package.json

```diff
@@ -51,13 +52,19 @@
     "electron": "electron .",
     "electron-dev": "cross-env ELECTRON_START_URL=\"http://localhost:3000\" electron .",
     "dev": "concurrently --kill-others \"npm:start\" \"sleep 10 && npm run electron-dev\"",
-    "dev:home": "cross-env REACT_APP_AT_HOME=\"true\" npm run dev",
-    "dev:clinic": "cross-env REACT_APP_AT_HOME=\"false\" npm run dev",
-    "build:clinic": "cross-env REACT_APP_AT_HOME=\"false\" npm run build",
-    "build:home": "cross-env REACT_APP_AT_HOME=\"true\" npm run build",
     "build:home:video": "cross-env REACT_APP_AT_HOME=\"true\" REACT_APP_VIDEO=\"true\" npm run build",
     "firebase:dev": "firebase emulators:start --import emulator_data",
-    "firebase:savedev": "firebase emulators:export emulator_data"
+    "firebase:savedev": "firebase emulators:export emulator_data",
+    "dev:home": "dotenv -e env/home.env npm run dev",
+    "dev:vid": "dotenv -e env/video.env npm run dev",
```

Comment:
do we really want dev versions for all of the iterations? also `vid` should be `video` to match the build script if so

Reply:
I think this just makes testing easier on our end

Reply:
I had the same thinking ^ we didn't originally have all the versions for dev, but I created some more so we could test.